install/kubernetes/cilium/templates/cilium-operator-deployment.yaml

```diff
@@ -154,7 +154,7 @@ spec:
 {{- else }}
             host: '::1'
 {{- end }}
-            path: /healthz
+            path: /v1beta/healthz
```

Comment:
Can we avoid changing this path?

Reply:
We can set the base path for open API spec to `/` and then we can avoid this change. What do you think should  be the preferred basePath for the cilium-operator API?

Reply:
Can't we have an alias path to also serve `/healthz`?

Reply:
One way I see is to initialize the `http.Server` using the default mux with a specific entry for `/healthz` and the default handler will fallback to open-api router.
I will check if this works.

Reply:
Seems to work

![image](https://user-images.githubusercontent.com/21292343/103642467-2e382280-4f79-11eb-93a1-dfc891227a9d.png)